scene/main/node.h

```diff
@@ -675,9 +678,15 @@ class Node : public Object {
 
 	_FORCE_INLINE_ Viewport *get_viewport() const { return data.viewport; }
 
+#ifndef DISABLE_DEPRECATED
 	virtual PackedStringArray get_configuration_warnings() const;
```

Comment:
All the inheriting methods also need the ifndef

Reply:
I assume you mean the nodes overriding/calling configuration warning stuff, like Label etc.?
I'll switch to the new methods everywhere as part of this PR, so that adding `DISABLE_DEPRECATED` shouldn't be needed anywhere else.

Reply:
Then yes assuming you replace them (but as you can see CI fails as there's methods in `Viewport` for example) but assuming you replace them it's sufficient